File: Diagonals sum.py
def sum_diagonals(matrix):
    if len(matrix[0])==0:
        return 0
    matrix_r = [tmp[::-1] for tmp in matrix]
    result = 0
    for i in range(0,len(matrix)):
        for j in range(0,len(matrix)):
            if i==j:
                result += matrix_r[i][j]
                result += matrix[i][j]
    return result

File: Diagonals sum/Diagonals sum.js
function sum(matrix) {
  const matrix_r = [...matrix].map(v => v.slice().reverse())
  let result = 0;
  for(let i=0;i<matrix.length;i++){
    for(let j=0;j<matrix.length;j++){
      if(i===j){
        result+=matrix[i][j]
        result+=matrix_r[i][j]
      }
    }
  }
  return result
}
